pyecharts/utils/__init__.py

```diff
@@ -45,16 +45,16 @@ def to_css_length(x):
         return x
 
 
-def _flat(ele):
+def _flat(obj):
     """
-    :param ele:
+    :param obj:
     :return: Return a list
     """
-    if hasattr(ele, 'js_dependencies'):
-        return list(ele.js_dependencies)
-    if isinstance(ele, (list, tuple, set)):
-        return ele
-    return ele,
+    if hasattr(obj, 'js_dependencies'):
+        return list(obj.js_dependencies)
+    if isinstance(obj, (list, tuple, set)):
+        return obj
+    return obj,  # tuple
```

Comment:
the last turn means we are returning it anyway. should you raise an exception here? tuple is detected in previous if. with the exception, the developer knows it during the development.

Reply:
`_flat` and `merge_js_dependencies` only accepts chart's dependencies as their input,which dependency building process should ensure.